apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -218,10 +218,18 @@ export default function ListBox({
   const isVertical = listLayout !== 'horizontal';
   const count = layout.qListObject.qSize.qcy;
 
-  const getAccumulatedHeight = (ps) => ps.reduce((acc, p) => acc + p.qArea.qHeight, 0);
-
-  const listCount = pages && pages.length && calculatePagesHeight ? getAccumulatedHeight(pages) : count;
+  const PAGE_MAX_HEIGHT = 100;
```

Comment:
I think this is from `minimumBatchSize={100}`.
consider also useing this constant as arguemnt to minimumBatchSize.
(as it is a minimun it could maybe be wring for a extremely large listbox. with more then 100 values visible at the same time)

Reply:
Yes, seems so. I will use that one instead.